sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/json/JsonFileFormat.scala

```diff
@@ -113,6 +113,18 @@ class JsonFileFormat extends TextBasedFileFormat with DataSourceRegister {
       }
     }
 
+    if (requiredSchema.length == 1 &&
+      requiredSchema.head.name == parsedOptions.columnNameOfCorruptRecord) {
+      throw new AnalysisException(
+        s"'${parsedOptions.columnNameOfCorruptRecord}' cannot be selected alone without other " +
```

Comment:
This line and the follow ling are concatenated and maybe too long. Add a `\n` in the end of this line?